saleor/order/tests/test_order_actions_create_fulfillments_for_returned_products.py

```diff
@@ -644,3 +644,80 @@ def test_create_return_fulfillment_with_lines_already_refunded(
     assert returned_and_refunded_fulfillment.shipping_refund_amount is None
 
     mocked_order_updated.assert_called_once_with(fulfilled_order)
+
+
+@patch("saleor.plugins.manager.PluginsManager.order_updated")
+@patch("saleor.order.actions.gateway.refund")
+def test_create_return_fulfillment_only_order_lines_with_old_ids(
```

Comment:
Lack of `given, when, then`. Also shouldn't we check in some tests with creating replacement order if `od_id` is not set?

Reply:
If possible let's fix that later, for now I'll merge this as it is.